packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -276,6 +276,15 @@ def ops_docker_compose_file(
     """
     docker_compose_path = Path(temp_folder / "ops_docker_compose.filtered.yml")
 
+    # these services are useless when running in the CI
+    ops_view_only_services = ["adminer", "redis-commander", "portainer", "filestash"]
+    if "CI" in os.environ:
+        ops_services_selection = list(

```

Comment:
MINOR: log the comment above in the CI: e.g. "Dropping services X,Y,Z from original stack for the CI runs"